### ISSUE TITLE
Default SmartGroupTemplate in JSS recipes

### ISSUE THREAD

**Issue by grahampugh** [mikaellofgren/RecipeBuilder#2]
The default Smart Group Template should be generic:
```
            <key>GROUP_TEMPLATE</key>
            <string>SmartGroupTemplate.xml</string>
```
Right now it writes in the name of the package, but there's no reason to do that with a default recipe:
```
            <key>GROUP_TEMPLATE</key>
            <string>Chrome-SmartGroupTemplate.xml</string>
```

**Comment by mikaellofgren** [comment]
Thank you, its been changed.

**Comment by mikaellofgren** [comment]
Fixed in 1.01
